gulpfile.babel.js

```diff
@@ -0,0 +1,48 @@
+import gulp from 'gulp';
+import jasmineNode from 'gulp-jasmine-node';
+import gls from 'gulp-live-server';
```

Comment:
The import name doesn't seem to be descriptive enough, can you compare this to other import names you used?